Makefile

```diff
@@ -32,21 +44,31 @@ endif
 ifneq ($(ATHENZ_SIA_DEFAULT_ROLE_CERT_EXPIRY_TIME_BUFFER_MINUTES),)
 LDFLAGS_ARGS += -X 'github.com/AthenZ/k8s-athenz-sia/pkg/identity.DEFAULT_ROLE_CERT_EXPIRY_TIME_BUFFER_MINUTES=$(ATHENZ_SIA_DEFAULT_ROLE_CERT_EXPIRY_TIME_BUFFER_MINUTES)'
 endif
+ifneq ($(ATHENZ_SIA_DEFAULT_INTERMEDIATE_CERT_BUNDLE),)
+LDFLAGS_ARGS += -X 'github.com/AthenZ/k8s-athenz-sia/pkg/identity.DEFAULT_INTERMEDIATE_CERT_BUNDLE=$(ATHENZ_SIA_DEFAULT_INTERMEDIATE_CERT_BUNDLE)'
+endif
 
 ifneq ($(LDFLAGS_ARGS),)
 LDFLAGS += -ldflags "$(LDFLAGS_ARGS)"
 endif
 
-build:
+build: submodule-update
 	@echo "Building..."
 	go mod tidy
 	CGO_ENABLED=0 go build $(LDFLAGS) -o $(GOPATH)/bin/athenz-sia cmd/athenz-sia/*.go
 
 test: build
 	@echo "Testing..."
 	go test ./...
+	CGO_ENABLED=1 go build $(LDFLAGS) -race -o $(GOPATH)/bin/athenz-sia cmd/athenz-sia/*.go
```

Comment:
Refactor with command in github action. Should use `make test` inside github action.
maybe no need for `go build -race`

Reply:
`go build -race` seems still necessary since go test does not cover for detecting race conditions

Reply:
`go build -race` just add the race detector inside the binary. Will update `make test` with `go test -race` for detecting race conditions